service/resource/kmskeyv2/error.go

```diff
@@ -15,3 +19,15 @@ var wrongTypeError = microerror.New("wrong type")
 func IsWrongType(err error) bool {
 	return microerror.Cause(err) == wrongTypeError
 }
+
+func IsKeyNotFound(err error) bool {
+	aerr, ok := err.(awserr.Error)
+	if !ok {
```

Comment:
nitpick, maybe we can remove this check and do later: `if ok && aerr.Code() == kms.ErrCodeNotFoundException` so that we have a single `return true` and a single `return false`?

Reply:
Done, that's much nicer. Thx!